server/datasources/requestTypeDataSource.ts

```diff
@@ -1,9 +1,44 @@
 import CachedMongooseDataSource from './cachedMongooseDataSource'
 import { RequestTypeCache } from '../database/cache'
 import { RequestTypeDocument } from '../models/requestTypeModel'
+import { ServerResponseInterface } from '../graphql/serverResponse'
+
+import { Types } from 'mongoose'
 
 export default class RequestTypeDataSource extends CachedMongooseDataSource<RequestTypeDocument> {
   constructor() {
     super(RequestTypeCache)
   }
+  
+  async createRequestType(requestType: RequestTypeDocument): Promise<ServerResponseInterface> {
```

Comment:
can't the user of this data source just call super.create directly?

Reply:
similar for the other functions

Reply:
hmm that's true. althought we don't have an instance of the cachedMongooseDataSource so i'm not sure how we would do that.